FIx styling

bryans99 opened the pull request:
Please check this out.

Will look at this more when I get to the other side :D

bryans99 committed:
FIx styling

Dutchgoose reviewed (approved):
Thanks for figuring this out Bryn

bryans99 committed:
1. Change to use @looker/components through out
2. Add styled-system (needed for margin, padding shortcuts and other things)
3. Changed dev server to use http (https not required)
4. Fixed readme references to https

bryans99 committed:
Updated README with information about Looker components

Dutchgoose commented on src/demo/QueryContainer.tsx:
```diff
@@ -24,23 +24,66 @@
+  Text,
+  TableDataCell,
+  TableBody
+} from "@looker/components"
```
Is there a way to import each one individually to reduce package size when using `yarn start`? It would be lame if the only time we have an optimized package is when we build.

(I may be totally misunderstanding how imports work here, feel free to educate me)

Dutchgoose replied:
Any word on this?

Dutchgoose reviewed (approved):
Thanks for Lensifying everything!

bryans99 committed:
1. Fix production build - issue with @types/styled-components bringing in react native stuff which causes a compiled error. Tied to version 4.1.8 which fixes issue for now but not ideal. Helltool has a similar issue which lens team fixed in a different way. Could not get there fix to work here but will go back and look at it later
2. Reduced size of build to 622k. Not great, but better than 900k. At some point will see if I can reduce. Might need to look at supporting code splitting sooner. At a minimum we should ensure browser caching is okay.
3. Tested compiled bundle.js  - seems to work

Dutchgoose reviewed (approved):
Looks good!
